x-pack/test/functional/services/ml/trained_models_table.ts

```diff
@@ -172,6 +172,20 @@ export function TrainedModelsTableProvider(
       );
     }
 
+    public async isModelCollapsedActionsButtonExists(modelId: string) {
```

Comment:
Nit: not exactly sure about our Kibana naming conventions, but `isXyzExists` looks a bit confused to me. Happy to leave it as is if that's matching what we have in other places.

Reply:
thanks for spotting this, it's simply grammatically incorrect. 😅 updated in https://github.com/elastic/kibana/pull/158533/commits/7c8ccb3fdb9da7c3a0e80be482161e9c87cf61da